src/components/Web3Status/index.tsx

```diff
@@ -239,11 +240,19 @@ function Web3StatusInner() {
           </RowBetween>
         ) : (
           <>
-            {hasSocks ? <Sock /> : null}
+            {hasSocks && !isNavbarEnabled ? <Sock /> : null}
```

Comment:
nit: Put the `!isNavbarEnabled` check first, as it's a more global check

    {!isNavbarEnabled && hasSocks ? <Sock /> : null}